### PR TITLE
put fixes on a new branch

### PR DESCRIPTION
**Pull request by TonyB9000**

### CODE REVIEW

**Comment by TonyB9000** [comment]
This branch (deleted locally) will NOT incur old code if the only changes are those shown here, right?

File: warehouse/warehouse/scripts/fix_mapfile_paths.py [hunk 1]
```diff
@@ -58,7 +59,8 @@ def main():
     mapfile_temp = str(mapfile_path.resolve())
     mapfile_path.unlink()
     move(tempfile.name, mapfile_temp)
-    os.chmod(mapfile_temp, 0o664)
+    mapfile_temp = path(mapfile_temp)
```
Comment (sterlingbaldwin) [inline warehouse/warehouse/scripts/fix_mapfile_paths.py:62]: this should be `Path()`

File: warehouse/warehouse/scripts/fix_mapfile_paths.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import sys
+import os
```
Comment (sterlingbaldwin) [inline warehouse/warehouse/scripts/fix_mapfile_paths.py:2]: since you switched to the pathlib method,  you should be able to remove this `os` import.

**Review by sterlingbaldwin — CHANGES_REQUESTED** [review]

File: warehouse/warehouse/scripts/fix_mapfile_paths.py (hunk 2 above)
Comment (TonyB9000) [reply warehouse/warehouse/scripts/fix_mapfile_paths.py:2]: Right.  I can do that and (remove os) and hopefully get Path correct.

**Comment by TonyB9000** [comment]
I believe this is already covered elsewhere.

**Pull request closed without merging**